keyboards/ergotravel/rev1/rev1.h

```diff
@@ -0,0 +1,37 @@
+#ifndef REV1_H
+#define REV1_H
+
+#include "ergotravel.h"
+
+//void promicro_bootloader_jmp(bool program);
+#include "quantum.h"
+
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+    #include <avr/io.h>
+    #include <avr/interrupt.h>
+#endif
+#endif
+
+//void promicro_bootloader_jmp(bool program);
+
+#define KEYMAP( \
```

Comment:
Please use `LAYOUT` here instead.  We're depreciating KEYMAP in favor of LAYOUT.

Reply:
Done